### PR TITLE
Complete the Toast component

### PR DESCRIPTION
**Pull request by KBhautik6**
### Toast section
Screenshot 2023-12-25 at 12 44 58 PM

- [x] Checking the code is duplicate or not?
- [x] Is these code has bug 
- [ ] Is code is check by shubhum sir or ankit sir or tara ma'am?
- [ ] Are you going to merge this or not?

### CODE REVIEW

File: src/stories/Toast.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import {StyledText} from "./Toast.styled"
+
+export const Toast=({text,colortype}) =>{
+  return (
+    <StyledText colortype={colortype}>
```
Comment (tarakhant1297) [inline src/stories/Toast.js]: @KBhautik6 rename colortype to color

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/stories/Toast.styled.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    display:flex;
+    gap:1rem;
+    background-color:${theme.offwhite};
+    justify-content:center;
+    flex-wrap:wrap;
+`
+export const StyledText=styled.p`
+    color:${({colortype})=>theme?.[colortype]};
+    background-color:${theme.white};
+    border:2px solid;
```
Comment (ankitadrixus08) [inline src/stories/Toast.styled.js]: use shorthand property here @KBhautik6

File: src/stories/Toast.js (hunk 1 above)
Comment (KBhautik6) [reply src/stories/Toast.js]: okay

File: src/stories/Toast.styled.js (hunk 2 above)
Comment (KBhautik6) [reply src/stories/Toast.styled.js]: okay

**Review by ankitadrixus08 — APPROVED** [review]

**Review by tarakhant1297 — APPROVED** [review]

**Pull request merged**